cmd/find/namePrimary.go

```diff
@@ -0,0 +1,28 @@
+package cmdfind
+
+import (
+	"fmt"
+
+	"github.com/gobwas/glob"
+	apitypes "github.com/puppetlabs/wash/api/types"
+)
+
+// namePrimary => -name ShellGlob
+var namePrimary = &atom{
```

Comment:
The parser makes it easy to create subsequent primaries and independently test them. I plan on adding a time attribute primary to handle `ctime`, `mtime` and `atime` after this PR's merged.